drivers/windows/R/batch.R

```diff
@@ -1,19 +1,48 @@
 write_batch_task_run <- function(task_id, config, path_root) {
+
+  run_on_linux <- config$platform == "linux"
+  template_file <- if (run_on_linux) "task_run.sh" else "task_run.bat"
+  out_script <- if (run_on_linux) SH_RUN else BATCH_RUN
+  write_os_lines <- if (run_on_linux) write_linux_lines else writeLines
```

Comment:
This might be easier to read in one if/else

```
if (run_on_linux) {
  template_file <- ...
  out_script <- ...
} else {
}
```

etc

Reply:
Refactored into batch-linux.R and batch-windows.R with batch.R doing the common stuff, so hopefully this looks nicer now.